src/NeoSharp.Core/Network/NetworkConfig.cs

```diff
@@ -25,6 +26,10 @@ public class NetworkConfig
         /// Acl Config
         /// </summary>
         public NetworkAclConfig AclConfig { get; internal set; }
+        /// <summary>
+        /// StandByValidator config
+        /// </summary>
+        public IEnumerable<string> StandByValidator { get; internal set; }
```

Comment:
string[] ?

Reply:
If you prefer, i can change to string[].
I generally prefer IEnumerable<T>